src/pages/MyTimePage/MyTimePage.tsx

```diff
@@ -55,6 +55,19 @@ export const MyTimePage = () => {
   const getDayComponent = (day: string) => {
     const colorIndex = -1;
     const freeHours = tasks[day]?.freeHrsOfDay;
+    
+    const today = new Date();
+    const dayOfMonth = today.getDate();
+    let suffix = "";
+    if (dayOfMonth === 1 || dayOfMonth === 21 || dayOfMonth === 31) {
```

Comment:
check whether the last character is equal to 1 or not. Avoid this approach of 3 checks

Reply:
Solved it using alternate approach